app/decorators/department_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class DepartmentDecorator < Draper::Decorator
+  delegate_all
```

Comment:
1. `head_of_d` is the unacceptable name of the method. Write `head_of_department` or just `head`. No contractions for a method, class, and file names.
2. this method should return only the head of the department, like an object, but not his `full_name`. I recommend creating of a separate method for it. `def full_name`.